suite/runsuite_wrapper.pl

```diff
@@ -360,6 +366,11 @@
                 'common.decode-stress' => 1, # i#1807 Ignored for all options.
                 'code_api,opt_speed|common.fib' => 1, # i#1807: Undiagnosed timeout.
                 'prof_pcs|common.nativeexec_exe_opt' => 1, # i#2052
+                'prof_pcs|common.nativeexec_retakeover_opt' => 1, # i#2052
+                'prof_pcs|common.nativeexec_bindnow_opt' => 1, # i#2052
+                'prof_pcs,thread_private|common.nativeexec_exe_opt' => 1, # i#2052
+                'prof_pcs,thread_private|common.nativeexec_retakeover_opt' => 1, # i#2052
```

Comment:
This one isn't present in the PR description with the thread_private option.

Reply:
Right,, like above, I extrapolated to cover similar failures in other arch/op combos to try and avoid future redness on variants. Will update the description on merge.